client/app/billing/autoRenew/actions/update/automatic/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "billing_autorenew_service_update_automatic_sumup": "Vous avez sélectionné le paiement automatique pour votre service {{ serviceName }}",
+  "billing_autorenew_service_update_automatic_frequency": "Ce service sera automatique renouvelé tous les {{ period }} à compter du {{ date }}",
```

Comment:
Isn't `semaine` possible ? Because it would be `toutes les semaines` then

Reply:
Might be worth checking but I think it's either hourly (pay as you go) or monthly